tests/pbft_manager_test.cpp

```diff
@@ -72,7 +72,7 @@ void check_2tPlus1_validVotingPlayers_activePlayers_threshold(size_t committee_s
   const auto init_bal = node_1_expected_bal / nodes.size();
   for (size_t i(1); i < nodes.size(); ++i) {
     auto master_boot_node_send_coins = std::make_shared<Transaction>(
-        nonce++, init_bal, gas_price, TEST_TX_GAS_LIMIT, bytes(), nodes[0]->getSecretKey(), nodes[i]->getAddress());
+        nonces[0]++, init_bal, gas_price, TEST_TX_GAS_LIMIT, bytes(), nodes[0]->getSecretKey(), nodes[i]->getAddress());
```

Comment:
- should be here nonces[i]++ ? 
- same applies for multiple occurences of nonces[0]++ below

Reply:
No. We should use nonce of sender(private key we specify). Maybe it is better to make some method\lambda for this like I did in other test

Reply:
ah ok, [optional]: maybe just create something like const auto sender_idx = 0 and use that, which would resolve the confusion